src/components/LayerSwitcher/SuccessLayerInput.tsx

```diff
@@ -0,0 +1,103 @@
+import { Autocomplete } from '@material-ui/lab';
+import {
+  Box,
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  TextField,
+} from '@material-ui/core';
+import * as React from 'react';
+import { Star } from '@material-ui/icons';
+import { LayerIndex } from './helpers/loadLayers';
+import { isViewInsideBbox } from './helpers';
+import { useMapStateContext } from '../utils/MapStateContext';
+
+type SuccessLayerDataInputProps = {
+  index: LayerIndex[];
+  onSelect: (layer: LayerIndex | undefined) => void;
+};
+
+const getLayerLabel = ({ name, country_code }: LayerIndex) => {
+  if (country_code) return `${country_code} - ${name}`;
+
+  return `Global - ${name}`;
+};
+
+export const SuccessLayerInput: React.FC<SuccessLayerDataInputProps> = ({
+  index,
+  onSelect,
+}) => {
+  const [showAllowedForOsm, setShowAllowedForOsm] = React.useState(true);
+  const [currentlyVisible, setCurrentlyVisible] = React.useState(true);
+  const { view } = useMapStateContext();
+
+  return (
+    <div>
+      <FormGroup>
+        <FormControlLabel
+          control={
+            <Checkbox
+              checked={showAllowedForOsm}
+              onChange={({ target: { checked } }) =>
+                setShowAllowedForOsm(checked)
+              }
+            />
+          }
+          label="Allowed for the use in osm"
```

Comment:
1) Please use a vocabulary.ts.
2) I suggest message "License compatible for editing OpenStreetMap"
3) Default=off (i think most users just consume the map). I will add activeLayers to changeset tags, so people don't abuse it by mistake.